src/templates/Footer/index.js

```diff
@@ -41,14 +41,18 @@ function Footer(props) {
     <div className={classnames(styles['footer'], className)}>
       <Layout
         className={styles['footer-layout']}
-        mediumCols={[ 5 ]}
+        mediumCols={[ 6 ]}
         smallCols={[ 12 ]}
         fullwidthAll
+        style={{ maxWidth: '1000px' }}
       >
-        <Col className={classnames(styles['footer-help'], styles['help'])}>
+        <Col
+          className={classnames(styles['footer-help'], styles['help'])}
+          style={{ maxWidth: '330px' }}
```

Comment:
@petrgazarov Not sure why you set this, but since this is actually in the RCL / Athenaeum you should do this in the scss. We only exposed this prop so that users can update it in their projects.